src/NexxLogic.BlobConfiguration.AspNetCore/Options/BlobConfigurationOptionsValidator.cs

```diff
@@ -4,14 +4,40 @@ namespace NexxLogic.BlobConfiguration.AspNetCore.Options;
 
 internal class BlobConfigurationOptionsValidator : AbstractValidator<BlobConfigurationOptions>
 {
-    public BlobConfigurationOptionsValidator()
+    public BlobConfigurationOptionsValidator(bool blobNameIsRequired)
```

Comment:
I am thinking on another approach, inheriting from BlobConfigurationOptionsValidator. Then, in derived constructor you can call the base to have the main validations then do the extra validation.

What do you think?